dynamodb-dump

```diff
@@ -24,29 +24,37 @@ current_dir=$(pwd)
 
 cd $(mktemp -d)
 
-file_num=0
-next_token=0
-
 dynamo_scan() {
-   result=""
-   
-   if [ -z "$1" ]
-   then
-      result=$(aws dynamodb scan --table-name $TABLE_NAME --max-items $NUM_ITEMS)
-   else
-      result=$(aws dynamodb scan --table-name $TABLE_NAME --max-items $NUM_ITEMS --starting-token $1)
-   fi
-
-   echo $result | jq -c '[{Item: .Items[]} | {PutRequest: .}]' | jq -c "{\"$TABLE_NAME\": .}" > $TABLE_NAME-$file_num
-
-   next_token=$(echo $result | jq .NextToken)
-}
+    result=""
+
+    if [ -z "$2" ]
+    then
+        result=$(aws dynamodb scan --table-name $1 --max-items $NUM_ITEMS)
+    else
+        result=$(aws dynamodb scan --table-name $1 --max-items $NUM_ITEMS --starting-token $2)
+    fi
 
-dynamo_scan
+    echo $result | jq -c '[{Item: .Items[]} | {PutRequest: .}]' | jq -c "{\"$var\": .}" > $var-$file_num
+
+    next_token=$(echo $result | jq .NextToken)
+}
 
-while [ "$next_token" != "null" ]; do
-   file_num=$((file_num+1))
-   dynamo_scan $next_token
+arg_num=0
+
+for var in "$@"
```

Comment:
I'm not sure how to start at 1... The reason we skip if arg_num <= 1 is because we want to ignore the first argument (the S3 path) in the `$@`